helm_deploy/values-dev.yaml

```diff
@@ -6,6 +6,14 @@ generic-service:
   service:
     enabled: true
 
+  resources:
+    requests:
+      cpu: 100m
+      memory: 1024Mi
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+
```

Comment:
Convention is for values.yaml to contain production values - so smaller limits go in the env-specific config